.github/workflows/snyk-monitor.yml

```diff
@@ -10,6 +10,11 @@ on:
       - '**/package.json'
```

Comment:
I just realized, does the snykfile path above need to be changed to `**/.snyk` now?

Reply:
yes 😅, threw it on the other PR